app/services/user/service.go

```diff
@@ -72,6 +72,8 @@ func (u *userService) SignUpUser(ctx context.Context, signUpParams *dto.UserSign
 		log.Infof("%v", err)
 		return nil, err
 	}
+	print(signUpParams.FirstName)
+	print(signUpParams.Email)
```

Comment:
Use log.InfoF if you want to actually log data, but here i'm assuming this is for debugging so remove it.